preprocessing/nextclade/tests/test_metadata_processing_functions.py

```diff
@@ -624,46 +549,6 @@ def test_preprocessing_without_consensus_sequences():
     assert processed_entry.data.aminoAcidInsertions == {}
 
 
-def test_format_frameshift():
```

Comment:
I moved these functions to the `test_nextclade_preprocessing` file as they are functions for the nextclade output